cnd/src/comit_api.rs

```diff
@@ -14,26 +14,26 @@ use std::str::FromStr;
 #[derive(Clone, Copy, derivative::Derivative, PartialEq)]
 #[derivative(Debug = "transparent")]
 pub enum LedgerKind {
```

Comment:
Doesn't need changing for this PR but I think we are overloading the term 'ledger', here we are referring to the network, right, perhaps `NetworkKind` would be better.  Also we started calling the protocol side 'ledger' also i.e., alpha/beta. I never commented on @thomaseizinger suggestion on slack but the more it sits with me the more I think 'side' might be a useful term. Anyways, just putting up some thoughts for later.

Reply:
What about just `Network` for this one?